components/new_channel_modal/new_channel_modal.jsx

```diff
@@ -110,13 +108,6 @@ export default class NewChannelModal extends React.PureComponent {
         this.displayNameInput = React.createRef();
     }
 
-    componentDidMount() {
-        // ???
-        if (UserAgent.isInternetExplorer() || UserAgent.isEdge()) {
-            $('body').addClass('browser--ie');
```

Comment:
No idea why the NewChannelModal was responsible for this, but this logic is duplicated by the ChannelController which is a much better place for it to go

Reply:
Should we pull this into a separate PR to keep the scope focussed on things for which we definitely don't need to have QA testing.

Reply:
I don't think it should be necessary for small  bits of cruft like this that shouldn't have any effect on the code. I'll make sure that the other jQuery removal PRs will include QA review when they actually have something to test though.